bootiso

```diff
@@ -772,11 +784,15 @@ mountISOFile() {
 # $1 : a device block
 # Return 0 if device is USB, 1 otherwise
 getDeviceType() {
-	typeset deviceName=/sys/block/${1#/dev/}
-	typeset deviceType=$(udevadm info --query=property --path="$deviceName" | grep -Po 'ID_BUS=\K\w+')
+	typeset deviceType=$(udevadm info --query=property "/dev/$device" | grep -Po 'ID_BUS=\K\w+')
```

Comment:
This line might introduce a bug.
I think you should use function argument `$1` instead of `$device` variable, shouldn't you ?

Also, `${1#/dev/}` can take `/dev/sda` **or** `sda` and turn it to `sda` ; which is handful when user select device with -d flag. 
For that reason, **be careful** with `$selectedDevice` global variable, as it can contain `/dev/sdX` kind of string. Perhaps we should normalize it's value at user param extraction phase.

Reply:
Reverted the change.